tensorboard/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Per https://github.com/kubeflow/manifests/issues/18

I  might suggest not porting over manifests for tensorboard, TFServing, TFJob, Katib, Seldon Deployment, until we figure out what we want to do here.